app/assets/stylesheets/application.css.scss

```diff
@@ -38,6 +38,11 @@ h2 {
   color: #ffffff;
 }
 
+ul {
+    list-style-type: none;
+    padding: 0;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered list-style-type, margin, padding